docs/install/local_setup.bash

```
@@ -0,0 +1,121 @@
+  _value="$2"
+
+  # get values from variable
+  eval _values=\"\$$_listname\"
+  # backup the field separator
+  _colcon_prefix_bash_prepend_unique_value_IFS="$IFS"
+  IFS=":"
+  # start with the new value
+  _all_values="$_value"
+  _contained_value=""
+  # iterate over existing values in the variable
+  for _item in $_values; do
```

Shellcheck (0.11.0)

[warning] _values is referenced but not assigned (did you mean '_value'?).

(SC2154)